example-app/SantokuApp/src/features/qa-home/pages/HomePage.tsx

```diff
@@ -36,23 +86,122 @@ export const HomePage: React.VFC<HomePageProps> = ({navigateToQuestionAndEventPo
     });
   }, [requestPermissionAndRegisterToken]);
 
+  // 利用規約に未同意の場合は、利用規約を表示します。
+  const showTermsAgreementOverlay = useShowTermsAgreementOverlay();
+  useFocusEffect(showTermsAgreementOverlay);
+
+  React.useLayoutEffect(() => {
```

Comment:
Why not import useLayoutEffect with destructuring?

Reply:
Fixed
5abb4b58d18ced8e6628eda6425dbabfcd5b644c